sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/query/ParallelDocumentQueryExecutionContext.java

```diff
@@ -185,13 +188,24 @@ private void initialize(
                 this.querySpec);
     }
 
-    private List<PartitionKeyRange> getPartitionKeyRangesForContinuation(
+    private void populatePartitionToContinuationMap(
+        Map<FeedRangeEpkImpl, CompositeContinuationToken> partitionMapping) {
+        for (Map.Entry<FeedRangeEpkImpl, CompositeContinuationToken> entry : partitionMapping.entrySet()) {
+            if (entry.getValue() != null) {
+                partitionKeyRangeToContinuationTokenMap.put(entry.getKey(), entry.getValue().getToken());
+            } else {
+                partitionKeyRangeToContinuationTokenMap.put(entry.getKey(), null);
```

Comment:
>null [](start = 76, length = 4)

named parameter

Reply:
done. Btw intellij automatically shows the names for unnamed nulls